projects/emissary-ingress/emissary/docker/linux/emissary-image/Dockerfile

```diff
@@ -0,0 +1,65 @@
+ARG BASE_IMAGE
+
+FROM public.ecr.aws/k1e6s8o8/eks-distro-minimal-base-python-builder:3.9-latest.2 as emissary-builder
```

Comment:
In your makefile add: 
```
BASE_IMAGE_REPO=public.ecr.aws/k1e6s8o8
BASE_IMAGE_NAME=eks-distro-minimal-base-python
```

Then add a file at the root of the repo `EKS_DISTRO_MINIMAL_BASE_PYTHON` with the value `3.9-latest.2`

Then in here just use the standard BASE_IMAGE and BUILDER_IMAGE vars.